.env.sample

```diff
@@ -11,7 +11,8 @@ JWT_KEY=
 SENDGRID_API_KEY=
 ADMIN_MAIL=
 PASSWORD=
-BASE_URL=
```

Comment:
why did you remove this?

Reply:
BASE_URL is repeated throughout the project for different routes, using it as the same name for the default SERVER URL, when being called on a similar page, it might cause a conflict